src/api/handlers/page_settings__home.py

```diff
@@ -42,15 +42,16 @@ def info(self, request):
       return MassenergizeResponse(error=str(err), status=err.status)
     return MassenergizeResponse(data=home_page_setting_info)
 
-  @admins_only
-  def publish(self, request):
-    context: Context = request.context
-    args: dict = context.args
-    home_page_id = args.pop('home_page_id', None)
-    home_page_setting_info, err = self.service.get_home_page_setting_publish(home_page_id)
-    if err:
-      return MassenergizeResponse(error=str(err), status=err.status)
-    return MassenergizeResponse(data=home_page_setting_info)
+  # this route isn't used - so not sure whether it works
+  #@admins_only
+  #def publish(self, request):
```

Comment:
it is used in the admin.  on the community profile page

Reply:
No, I do not think it is used.  communities.update is used, and works fine for that purpose.  I don't see a reason to keep an additional routine to be maintained for the same purpose.